nucypher/cli/commands/contacts.py

```diff
@@ -0,0 +1,148 @@
+"""
+ This file is part of nucypher.
+
+ nucypher is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ nucypher is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
+"""
+
+import click
+import os
+import shutil
+
+from nucypher.characters.control.emitters import StdoutEmitter
+from nucypher.cli.actions.select import select_card
+from nucypher.cli.options import option_force
+from nucypher.cli.painting.policies import paint_single_card, paint_cards
+from nucypher.cli.types import EXISTING_READABLE_FILE, UMBRAL_PUBLIC_KEY_HEX
+from nucypher.policy.identity import Card
+
+id_option = click.option('--id', 'card_id', help="A single card's checksum or ID", type=click.STRING, required=False)
+type_option = click.option('--type', 'character_flag', help="Type of card: (A)lice or (B)ob.", type=click.STRING, required=False)
+verifying_key_option = click.option('--verifying-key', help="Alice's or Bob's verifying key as hex", type=click.STRING, required=False)
+encrypting_key_option = click.option('--encrypting-key', help="An encrypting key as hex", type=click.STRING, required=False)
+nickname_option = click.option('--nickname', help="Human-readable nickname / alias for a card", type=click.STRING, required=False)
+
+
+@click.group()
+def contacts():
+    """Lightweight contacts utility to store public keys of known ("character cards") Alices and Bobs."""
+
+
+@contacts.command()
+@click.argument('query')
+@click.option('--qrcode', help="Display the QR code representing a card to the console", is_flag=True, default=None)
+def show(query, qrcode):
+    """View existing character card"""
```

Comment:
```bash
 [58] → nucypher contacts show --help
Usage: nucypher contacts show [OPTIONS] QUERY

  View existing character card

Options:
  --qrcode  Display the QR code representing a card to the console
  --help    Show this message and exit.
```

Consider clarifying what `QUERY` should be.
Example:
```suggestion
    """
    View existing character card.
    
    QUERY can either be the card id or nickname.
    """
```

Reply:
:heavy_check_mark: